apps/console/src/public/deployment.config.json

```diff
@@ -149,7 +149,8 @@
                 "id": "QWNjb3VudERpc2FibGU",
                 "displayName": "Account Disabled"
             }
-        ]
+        ],
+        "outboundProvisioningConnectors": []
```

Comment:
Let's add the corresponding j2 update and send a pr to framework as well.

Reply:
That support is already there.